src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <div class="card__info">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__rate">
+        <div class="card__rate-stars">
+          <div class="stars stars--4 stars--card">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="card__rate-count">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-title">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+      <button class="card__buy-button">
+        <a
+          href="#"
+          class="card__buy-button-link"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </button>
```

Comment:
Using a `<button>` element to wrap an `<a>` tag is not semantically correct. A button should not contain interactive elements like links. Consider using either a `<button>` with an `onclick` event or styling the `<a>` tag to look like a button.